packages/account/src/test-utils/launchNode.ts

```diff
@@ -217,6 +216,8 @@ export const launchNode = async ({
       snapshotDirToUse = tempDir;
     }
 
+    const { spawn } = await import('child_process');
```

Comment:
This dynamic import is necessary in order for `vitest` to not necessitate a polyfill for it.